tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "declaration": true,
+    "removeComments": false,
+    "noImplicitAny": true,
+    "outDir": "built/"
+  },
+  "exclude": [
+    "typings",
+    "node_modules",
+    "testapp"
+  ],
+  "files": [
```

Comment:
don't include both exclude and files.

https://github.com/Microsoft/TypeScript/wiki/tsconfig.json#details says
The "files" property cannot be used in conjunction with the "exclude" property. If both are specified then the "files" property takes precedence.

files will probably get you into trouble when you have to list every single test file, so just use exclude and let the compiler find all `.ts` files under this directory.